### ISSUE TITLE
Filter by location

### ISSUE THREAD

**Issue by dwcaraway** [18F/afsmallbiz#23]
A user may limit search results to a certain geographic area.

**Comment by arowla** [comment]
For this and the other filtering tasks, we should collaborate and see if it makes sense to add some standard parameters to the API wrapper layer for FBOpen to accomplish this... especially as we transition between backends.

**Comment by dwcaraway** [comment]
Agree. For elasticsearch and geospatial boundaries, was looking at elastic.js [GeoBboxFilter](http://docs.fullscale.co/elasticjs/ejs.GeoBboxFilter.html)

Would like to have a common way of performing the queries.

**Comment by arowla** [comment]
Wouldn't this require reliable geocoding on opportunities?

**Comment by dwcaraway** [comment]
it would. For FedBizOps solicitations, an address is provided which could be geocoded. For grants.gov and SBIRs, physical address is not provided. This feature was requested but may be difficult to implement in practice.

That said, a partial solution may be better than no solution. FedBizOps does currently offer filtering by place of performance.

**Comment by aaronsnow** [comment]
As @arowla alluded to, unfortunately the address info in FedBizOpps is
spotty and not at all normalized. There may be something clever we can do
to improve the data with some smart automated scrubbing.

On Tue, Apr 15, 2014 at 2:57 PM, Dave Caraway notifications@github.comwrote:

> it would. For FedBizOps solicitations, an address is provided which could
> be geocoded. For grants.gov and SBIRs, physical address is not provided.
> This feature was requested but may be difficult to implement in practice.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/18F/afsmallbiz/issues/23#issuecomment-40520169
> .

## 

Aaron Snow
Presidential Innovation Fellow
aaron.snow@gsa.gov
202-631-4667
